Prace_domowe/Praca_domowa2/kinga_ulasik/PD2_Kinga_Ulasik.ipynb

```diff
@@ -0,0 +1,1641 @@
+{
```

Comment:
<p>Target encoding to nie jest dowolna metoda bazująca na targecie tylko średnia w podziale na kategorie</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>